### ISSUE TITLE
Bug - Menu/Menu toggle - clean up guidance on aria-label usage

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly#5198]
Right now, the Menu docs read as below:
![Screen Shot 2022-10-28 at 2 47 51 PM](https://user-images.githubusercontent.com/26305082/198710211-f14bf7c4-af38-4b85-a648-b175c4d57ef7.png)

In fact, whether or not there is a favorite modifier on the action, there should always be an aria-label on a pf-c-menu__item-action-icon because there is no visible descriptive text labelling the icon.

Additionally, right now the MenuToggle docs include this usage of `aria-label`:
![Screen Shot 2022-10-28 at 3 47 49 PM](https://user-images.githubusercontent.com/26305082/198720456-22dc3931-acad-49c4-b41c-f753310d384d.png)

It's not the fact that `pf-m-plain` modifier is on the MenuToggle that requires the addition of an `aria-label`. Instead, it can be updated to note that an aria-label should be placed on the `pf-c-menu-toggle` whenever there is no text visible in the menu toggle. Often, plain modifiers are used when the menu toggle only contains an icon.

**Comment by mcoker** [comment]
closed via https://github.com/patternfly/patternfly/pull/5483
